src/wp-includes/post.php

```diff
@@ -7660,36 +7658,14 @@ function wp_queue_posts_for_term_meta_lazyload( $posts ) {
 					// Backward compatibility for if a plugin is putting objects into the cache, rather than IDs.
 					foreach ( $_term_ids as $term_id ) {
 						if ( is_numeric( $term_id ) ) {
-							$prime_term_ids[]                  = (int) $term_id;
-							$prime_taxonomy_ids[ $taxonomy ][] = (int) $term_id;
+							$term_ids[] = (int) $term_id;
 						} elseif ( isset( $term_id->term_id ) ) {
-							$prime_taxonomy_ids[ $taxonomy ][] = (int) $term_id->term_id;
-							$prime_term_ids[]                  = (int) $term_id->term_id;
+							$term_ids[] = (int) $term_id->term_id;
 						}
 					}
 				}
 			}
 		}
```

Comment:
```suggestion
		}
		$term_ids = array_unique( $term_ids );
```
Remove duplicate teem ids from the array.